demo/src/main/java/com/project/demo/entity/LoginResponse.java

```diff
@@ -9,6 +9,15 @@ public class LoginResponse {
 
     private long expiresIn;
 
+    private Integer user_id;
```

Comment:
Please consider to use camel case format.

Reply:
Iván already refactored it. Fixing comment.